assets/js/modules/pagespeed-insights/index.js

```diff
@@ -38,24 +41,10 @@ addFilter(
 	fillFilterWithComponent( PageSpeedInsightsSettings )
 );
 
-const {
-	active,
-	setupComplete,
-} = getModulesData()[ 'pagespeed-insights' ];
-
-if ( active && setupComplete ) {
-	// Add to main dashboard.
-	addFilter(
-		'googlesitekit.DashboardModule',
-		'googlesitekit.PageSpeedInsights',
-		createAddToFilter( <DashboardSpeed /> ),
-		45
-	);
-	// Add to dashboard-details view.
-	addFilter(
-		'googlesitekit.DashboardDetailsModule',
-		'googlesitekit.PageSpeedInsights',
-		createAddToFilter( <DashboardSpeed /> ),
-		45
-	);
-}
+domReady( () => {
+	global.googlesitekit.widgets.registerWidget( 'pagespeedInsightsWebVitals', {
```

Comment:
Let's use imports to reference these rather than globals as well, to make future maintenance easier. The IB is a bit ambiguous about global usage for this one, but we can use imports to get at this function instead and for our own code I think that's a better approach.

Reply:
Updated.